airbyte-webapp/src/pages/routes.tsx

```diff
@@ -142,6 +144,15 @@ export const Routing = () => {
     }
   }, [workspace]);
 
+  const customer = {
+    external_id: workspace.customerId
```

Comment:
(you can also add `name`, `email`, and freeform `metadata` here if you'd like :) e.g. https://github.com/papercups-io/chat-widget#using-in-react)